cmd/init.go

```diff
@@ -32,7 +32,7 @@ var initCmd = &cobra.Command{
 	Use:   "init",
 	Short: "Init sidekick CLI and configure your VPS to host your apps",
 	Long: `This command will run you throgh the setup steps to get sidekick loaded on your VPS.
-		You wil neede to provide your VPS IPv4 address and a registery to host your docker images.
+		You wil neede to provide your VPS IPv4 address and a registry to host your docker images.
```

Comment:
I think this one is already fixed by the other PR. Could you please remove this? I'll merge the other change!

Reply:
Removed the init.go change. Left utils.go in tact. Thanks!